### DISCUSSION TITLE
[FEATURE] Anlagen-Workflow

### DISCUSSION THREAD

**Discussion by stm9x9** [jlawyerorg/j-lawyer-org#2657] [Ideas]
nur als perspektivische Idee ein Fundstück aus einer Diskussion, vielleicht kann sowas ähnliches auch mal implementiert werden. Wobei ich aktuell keine Ahnung habe, wie das innerhalb des jl/LO-Ökosystem geht:

"In RA-Micro füge ich eine eine Anlage aus der Akte wortwörtlich mit ganzen zwei Klicks in den Fließtext in Word ein und übergebe hinterher dann das fertige Schreiben mit weiteren zwei Klicks an die beA-Schnittstelle. Die Anlagen werden dann automatisch korrekt benannt (Anlage K1, K2 etc) und auch angepasst, wenn sich die Reihenfolge im Fließtext noch ändert, mit einem Anlagenstempel versehen, und Sonderzeichen im Dateinamen werden automatsich ersetzt. Am Ende des Tages muss ich dann nur noch mit weiteren 2 Klicks den Postausgang öffnen und auf Senden drücken, um alle Schreiben des Tages durchs Netz zu jagen. "

**Comment by j-dimension** [comment]
Ich kenne die Anwendung nicht, aber die Wortmeldung hört sich doch sehr nach Fanboyism an. Das Stempeln wäre allerdings ein feines Feature.

In dieser Form ist mir das Ticket zu unscharf. Ich verschiebe in die Diskussionen.
